internal/data/spicedb_test.go

```diff
@@ -117,6 +118,16 @@ func TestSecondCreateRelationshipSucceedsWithTouchTrue(t *testing.T) {
 	assert.True(t, exists)
 }
 
+func TestIsBackendAvailable(t *testing.T) {
```

Comment:
If you want to test a case where SpiceDB isn't available, you could call the repository's constructor explicitly with incorrect values (ex: the wrong port number)

Reply:
Issue i've been experiencing with this is that we have a check in `NewSpiceDbRepository()` that tests a connection to spicedb and errors out if it fails. So having an invalid port fails the whole thing before we can test our IsBackendAvailable() func.

Reply:
..Is it this?: https://github.com/project-kessel/relations-api/blob/main/internal/data/spicedb.go#L69

Reply:
Yes, sorry not linking

Reply:
All good. That one's my bad- I added it locally when trying to reproduce the e2e test issues and didn't mean to push it. I just opened a PR to take it out.

Reply:
Oh hey, Raj merged it, so that should be out of your way after you rebase.